### PR TITLE
Add missed links for updatenode and config dhcp

### PR DESCRIPTION
**Pull request by caomengmeng**
1. Updatenode
Link： https://xcat-docs.readthedocs.org/en/latest/guides/admin-guides/manage_clusters/ppc64le/updatenode.html
Description:
Item1: For "Using_Clone_to_Deploy_Server#Update_Nodes_Later_On " missed link.
Here is the content: https://xcat-docs.readthedocs.org/en/latest/advanced/sysclone/sysclone.html
Item2: For "Use instuctions in Sync-Files-label" missed link.
Here is the content: https://xcat-docs.readthedocs.org/en/latest/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.html

2. Config DHCP
Link: https://xcat-docs.readthedocs.org/en/latest/advanced/hierarchy/configure_dhcp.html
Description: For "XCAT_pLinux_Clusters/#setup-dhcp“ missed link.
Here is the content: https://xcat-docs.readthedocs.org/en/latest/guides/admin-guides/manage_clusters/ppc64le/discovery/config_environment.html

### CODE REVIEW

**Comment by caomengmeng** [comment]
@bybai  Would you please help review the issue1, updatenode content is linked right or not?
@chenglch Would you please help review the issue2, config DHCP content is linked right or not?

File: docs/source/advanced/hierarchy/configure_dhcp.rst [hunk 1]
```diff
@@ -1,8 +1,7 @@
 Configure DHCP
 ==============
 
-Add the relevant networks into the DHCP configuration, refer to:
-`XCAT_pLinux_Clusters/#setup-dhcp <http://localhost/fake_todo>`_
+Add the relevant networks into the DHCP configuration, refer to: :ref:`Setup-dhcp`
```
Comment (chenglch) [inline docs/source/advanced/hierarchy/configure_dhcp.rst]: Thanks,  this link is right.

**Comment by bybai** [comment]
@caomengmeng  
1, `Setting up syncfile` section can link to doc ``The synclist file``; 
2, you can add more sysclone descriptions in this line`See Update Nodes Later On for more information` , since `Update Nodes Later On` is in sysclone doc.

**Comment by bybai** [comment]
Thanks,  these links are right. let me merge it.

**Comment by whowutwut** [comment]
@bybai  and @chenglch , can you help Meng Meng configure her Git environment.  I would not have allowed the pull request to be merged since the commits were done using a "root" user.  This means there is no identity associated with the changes. 

image

We need to be careful about this. You can have your git in a development machine, but set the git config to be your own user and identity.

**Comment by whowutwut** [comment]
@caomengmeng ,  Here's a danger of doing a ``git commit -a`` 
If you look at the files changes, it included a .swp file.  

I'm going to remove it in git manually, just FYI for future reference...

**Comment by whowutwut** [comment]
Also, at config_environment.rst:22. the following warning is created....

source/guides/admin-guides/manage_clusters/ppc64le/discovery/config_environment.rst:22: WARNING: duplicate label setup-dhcp, other instance in xcat-core/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/switch_discovery.rst

**Pull request merged**